.github/workflows/test.yaml

```diff
@@ -204,6 +238,7 @@ jobs:
       - "unit-tests"
       - "integration-tests-dbless"
       - "integration-tests-postgres"
+      - "integration-tests-enterprise-postgres"
```

Comment:
Question, not a blocker: should we do a DBLESS test too? We've seen issues that only happen in one of the backend modes, which is why we do the separate tests for OSS. My assumption is this PR leaves out DBLESS because we have high confidence that would end up being redundant, is that accurate?